port/unix/omrsignal.c

```diff
@@ -89,6 +89,15 @@ static struct {
  */
 static uint32_t signalsWithHandlers;
 
+/* Records the (port library defined) signals for which we have registered a master
```

Comment:
Avoid using "we"
for which we have registered a master handler -> for which a master handler is registered.
then we should avoid -> then avoid